script.js

```diff
@@ -0,0 +1,44 @@
+const loadMovieInfo = (name) => {
+    if (!name) { //This is runs when selected option is blank
+        document.querySelector('#movie_info').innerHTML = `<h3>To begin select a movie</h3>`;
+        document.querySelector('form').style.display = 'none';
+        return; //Prevents the rest of function from running
+    }
+    let filmObj = filmsObject[name];
+    let title = document.createElement('h3');
+    let year = document.createElement('p');
+    let description = document.createElement('p');
+    let filmArr = [title, year, description];
+    
+    title.innerText = filmObj.title;
+    year.innerText = filmObj.release_date;
+    description.innerText = filmObj.description;
+    filmArr = [title, year, description];
+    document.querySelector('form').style.display = 'block';
+    document.querySelector('#movie_info').innerText = '';
+    filmArr.forEach(item => document.querySelector('#movie_info').appendChild(item));
+}
+
+const addReview = () => {
+    let review = document.createElement('li');
+    review.innerHTML = `<strong>${document.querySelector('h3').innerText}:</strong> ${document.querySelector('input').value}`;
```

Comment:
A line like this is hard to process for a reviewer of your code. It would have been better to create variables that hold the results of the `.querySelector()` calls (like you did on `index.js:8-10`) and then use those for the interpolation here.